logical/framework/openapi.go

```diff
@@ -618,7 +618,8 @@ type cleanedResponse struct {
 	Data     map[string]interface{}     `json:"data,omitempty"`
 	Redirect string                     `json:"redirect,omitempty"`
 	Warnings []string                   `json:"warnings,omitempty"`
-	WrapInfo *wrapping.ResponseWrapInfo `json:"wrap_info,omitempty"`
+	WrapInfo *wrapping.ResponseWrapInfo `json:"wrap_info,omitempty" mapstructure:"wrap_info"`
```

Comment:
Why was the 'mapstructure' tag added?

Reply:
mapstructure can't automatically deduce "WrapInfo" from "wrap_info". But, I'm going to remove the mapstructure dependency here, so this will become a moot point.